tests/functions/test_drop_table_cascade.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+
+@unittest.skip("No tests has yet been implemented.")
+# This file should test the function "drop_table_cascade" in atc/functions
```

Comment:
maybe add a comment that our current test pipeline does not support delta tables yet.
Also in the other skipped test.

Reply:
I have rewritten the comment (y).